.github/workflows/build-and-push-assets.yml

```
@@ -48,6 +48,26 @@ on:
         default: './assets'
         required: false
         type: string
+      BUILT_BRANCH_SUFFIX:
+        description: Suffix for branch creating for compiled assets (`-built` is recommended for convention).
```

Can we use the description from the documentation?
```suggestion
        description: Suffix to calculate the target branch for pushing assets on the `branch` event.
```